packages/schematics/src/app/npmjs-utils.ts

```diff
@@ -0,0 +1,31 @@
+import { get } from 'http';
+
+export interface NpmRegistryPackage {
+  name: string;
+  version: string;
+}
+
+export function getLatestNodeVersion(packageName: string): Promise<NpmRegistryPackage> {
```

Comment:
The import I showed you might also do this too.

Reply:
I couldn't find this function, they seem to hard-code the latest Angular versions (I guess during their build process).